### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ashkan-o**
just solved a grammatical problem

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ ElSelect('p').AddLorem({
 ```
 *After select the element you can add your lorem text with **AddLorem()** and the options*
 
-Another Functions :
+There are two other Functions :
```
Comment (parsa011) [inline README.md:60]: in future may will add some other functions ! i think this is not very interesting for the this section title

**Review by parsa011 — COMMENTED** [review]
about the title

File: README.md (hunk 1 above)
Comment (ashkan-o) [reply README.md:60]: No problem but 'Another' is related to just one other thing, If there are more things, we use 'other'. you should solve that, you can do it in another way.

Comment (parsa011) [reply README.md:60]: Thanks for your suggestion

**Pull request closed without merging**